client.py

```diff
@@ -105,11 +107,26 @@ def send_model():
 # mqtt stuff
 #########################################
 
+
+def send_client_id():
+    global DEVICE_TOPIC
+    message = {
+        "message": PI_ID
+    }
+    message = json.dumps(message)
```

Comment:
nit: unnecessary because send_typed_message already checks if not a json and does the json dump.